src/components/common/request-details.vue

```diff
@@ -104,12 +108,16 @@ export default class RequestDetails extends Vue {
 <style lang="scss" scoped>
 @import "@/assets/scss/theme";
 
-h4 {
+.label {
   font-weight: bold !important;
   font-size: 1rem;
   color: $dk-text;
 }
 
+h4 {
+  @extend .label
+}
+
```

Comment:
Did these not exist globally? Seems very generic!

Reply:
yes, I see that there's a class in base.scss that has those properties, then I changed to that other class. Thanks.